backend/src/db-utils.js

```diff
@@ -568,20 +570,24 @@ const queryAccountOutcomeTransactionsCount = async (accountId) => {
     const query = await querySingleRow(
       [
         `SELECT SUM(outgoing_transactions_count) AS out_transactions_count,
-                MAX(CAST(EXTRACT(EPOCH FROM DATE_TRUNC('day', collected_for_day + INTERVAL '1 day')) AS bigint) * 1000 * 1000 * 1000) AS last_day_collected_timestamp
+                MAX(collected_for_day) AS last_day_collected
          FROM daily_outgoing_transactions_per_account_count
          WHERE account_id = :account_id`,
         { account_id: accountId },
       ],
       { dataSource: DS_ANALYTICS_BACKEND }
     );
+    const lastDayCollectedTimestamp = query?.last_day_collected
+      ? (new Date(query.last_day_collected).getTime() +
+          ONE_DAY_TIMESTAMP_MILISEC) *
+        1000 *
+        1000
```

Comment:
This overflows JS number type since the timestamp in nanoseconds is ~2^60, but JS can only keep the precision for the integers up to 2^53:

```
> Math.pow(2, 53) + 1
9007199254740992
> Math.pow(2, 53)
9007199254740992
```

Use BN.js